commands.json

```diff
@@ -2,7 +2,7 @@
     "TFCALL": {
         "summary": "Invoke a function",
         "since": "2.0.0",
-        "group": "gears",
+        "group": "triggersandfunctions",
```

Comment:
@thomascaudron , RedisInsight will display exactly "triggersandfunctions"
we can't merge this

Reply:
I had no idea that other teams used the commands.json files. Let me give this some thought and talk to David M.